src/gdcmodels/export/cli.py

```diff
@@ -0,0 +1,164 @@
+"""The basic cli functionality for the export."""
```

Comment:
Would you please add the context on when this is needed to be run?

Reply:
I tend to not favor duplicating documentation as then it just becomes out of sync. Also I tried to just detail the general purpose of the module here and [here](https://github.com/NCI-GDC/gdc-models/blob/1d3f291309651cbbcce0a4c9e98fefb129f16f06/README.md#examples) in the README, I've included the when and how to run the export.